examples/mnist.py

```diff
@@ -84,5 +89,7 @@ def main(save_to, num_epochs):
     parser.add_argument("save_to", default="mnist.pkl", nargs="?",
                         help=("Destination to save the state of the training "
                               "process."))
+    parser.add_argument("--no-bokeh", type=bool, default=False,
```

Comment:
I think you can set the default to `True`, or better, change the argument to `--bokeh`. The main reason we have Bokeh in here is for testing.